add-custom-dynamic-entity-data-source/src/main/java/com/esri/arcgismaps/sample/addcustomdynamicentitydatasource/screens/MainScreen.kt

```diff
@@ -74,7 +77,28 @@ fun MainScreen(sampleName: String) {
                         .weight(1f),
                     mapViewProxy = mapViewModel.mapViewProxy,
                     arcGISMap = mapViewModel.arcGISMap,
-                    onSingleTapConfirmed = mapViewModel::identify
+                    onSingleTapConfirmed = mapViewModel::identify,
+                    content = {
+                        mapViewModel.selectedGeoElement?.let { selectedGeoElement ->
+                            Callout(
+                                modifier = Modifier.wrapContentSize(),
+                                geoElement = selectedGeoElement,
+                                // Optional parameters to customize the callout appearance.
+                                shapes = CalloutDefaults.shapes(
+                                    calloutContentPadding = PaddingValues(
+                                        4.dp
+                                    )
+                                ),
+                                colorScheme = CalloutDefaults.colors(MaterialTheme.colorScheme.background, MaterialTheme.colorScheme.primary)
```

Comment:
Formatting suggestion and use of named arguments here: 
Also for the `borderColor` Compose material theme provides an `outline` Color which may be better for the Callout
```suggestion
                                shapes = CalloutDefaults.shapes(
                                    calloutContentPadding = PaddingValues(4.dp)
                                ),
                                colorScheme = CalloutDefaults.colors(
                                    backgroundColor = MaterialTheme.colorScheme.background,
                                    borderColor = MaterialTheme.colorScheme.outline
                                )
```